src/pages/BirthDay.jsx

```diff
@@ -12,16 +14,14 @@ const BirthDay = () => {
           <Typography fontSize="20px">생일!</Typography>
         </TitleWrapper>
         <MonthChip>
-          <Typography>MonthChip</Typography>
+          <ChipsArray />
```

Comment:
```suggestion
          chipArr.map((item) => <Chip>{item.label}</Chip>
```
이런식으로 데이터를 담은 배열에 따라 컴포넌트를 렌더링해야함.